src/setupRtl.test.tsx

```diff
@@ -28,4 +29,24 @@ describe("setupRtl", () => {
 
     view.getByText(text);
   });
+
+  describe("enforces that required props that are missing in the initial setup are provided in the render method", () => {
```

Comment:
There are no changes to `setupRtl`. Did you mean to add `RenderRtlReturn` to it similar to `setupEnzyme`'s changes?

Reply:
Ah, yeah. While I was re-cloning the repo to figure out why I didn't have permissions to PR, I must've lost this file's changes. Updated.